src/main/java/com/booleanuk/core/Basket.java

```diff
@@ -1,5 +1,67 @@
 package com.booleanuk.core;
 
+import java.util.ArrayList;
+
 public class Basket {
 
+    ArrayList<String> items;
+    int basketSize = 10;
+
+
+    public Basket(){
+
+       this.items = new ArrayList<>(basketSize);
+       items.add("Regular");
```

Comment:
I would avoid hardcoding in items to go in the list, instead add and remove them in the tests.